src/main/java/org/folio/circulation/services/LostItemFeeChargingService.java

```diff
@@ -196,15 +211,14 @@ private Result<List<CreateAccountCommand>> buildAccountsAndActions(
       }
 
       if (policy.getDeclareLostProcessingFee().isChargeable()) {
-        log.debug("Charging lost item processing fee");
 
+        log.debug("Charging lost item processing fee for set or actual costs");
```

Comment:
Should the log message be reverted as well?

Reply:
Actually it is valid for both scenarios.